sw-emulator/app/Cargo.toml

```diff
@@ -9,10 +9,14 @@ edition = "2021"
 
 [dependencies]
 clap = { version = "3.2.14", default-features = false, features = ["std"] }
+caliptra-registers = { path = "../../registers" }
 caliptra-emu-bus = { path = "../lib/bus" }
 caliptra-emu-cpu = { path = "../lib/cpu" }
 caliptra-emu-periph = { path = "../lib/periph" }
 caliptra-emu-types = { path = "../lib/types" }
+caliptra-hw-model = { path = "../../hw-model" }
```

Comment:
Do we have any risk for circular dependencies? emu's peripheral lib is a dependency of hw-model crate's emulated model. Should we aim for an emulator app to be built on top of hw-emulated-model

Reply:
This is a binary, so it can't be a library dependency (and thus no risk of circular deps).

Regardless, eventually I'd like to either delete this app or re-implement it on top of HwModel and move it somewhere inside hw-model/.